server/memcached/src/test/java/org/infinispan/server/memcached/text/MemcachedTextBigBlobTest.java

```diff
@@ -0,0 +1,32 @@
+package org.infinispan.server.memcached.text;
+
+import static org.testng.AssertJUnit.assertTrue;
+
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+import org.infinispan.server.memcached.configuration.MemcachedProtocol;
+import org.infinispan.server.memcached.test.MemcachedBigBlobTest;
+import org.testng.annotations.Test;
+
+@Test(groups = "functional", testName = "server.memcached.text.MemcachedTextBigBlobTest")
+public class MemcachedTextBigBlobTest extends MemcachedBigBlobTest {
+
+   @Override
+   protected MemcachedProtocol getProtocol() {
+      return MemcachedProtocol.TEXT;
+   }
+
+   public void testBufferOverflowCausesUnknownException() throws Exception {
```

Comment:
Have to admit this name of a test is really confusing... we aren't actually checking for an unknown exception afaic see.

Reply:
Yeah. The test name really does not match the actual body and I am unsure about the goal.

Reply:
It seems this exists because of the previous implementation. I'll drop it.